heapster-saw/src/Verifier/SAW/Heapster/UntypedAST.hs

```diff
@@ -10,6 +10,7 @@ data AstFunPerm = AstFunPerm
   Pos
   [(Located String, AstType)]
   [(Located String, AstExpr)]
+  [(Located String, AstType)]
   [(Located String, AstExpr)]   -- ^ @-o@ position, context, inputs, outputs
```

Comment:
I believe this comment is not longer correctly placed? I think it would also be worth adding comments to all of these fields saying what they are, since I find these couple lines very confusing.

Reply:
Ok, I see what you mean